src/libs/OptionsListUtils.ts

```diff
@@ -662,6 +662,20 @@ function isSearchStringMatchUserDetails(personalDetail: PersonalDetails, searchV
     return isSearchStringMatch(searchValue.trim(), memberDetails.toLowerCase());
 }
 
+/**
+ * Get IOU report ID of report last action if the action is report action preview
+ */
+function getIOUReportIDOfLastAction(report: OnyxEntry<Report>): string | undefined {
+    if (!report?.reportID) {
+        return;
+    }
+    const lastAction = visibleReportActionItems[report.reportID];
```

Comment:
NAB: Can we rename `visibleReportActionItems` to `lastVisibleReportActions`? Not a blocker since it's been named that way for a while. You can create a follow up PR for this and tag me for review.